app/assets/stylesheets/modules/_compliments.scss

```diff
@@ -0,0 +1,98 @@
+.send {
+  border: 1px solid $white;
+  float: right;
+  padding: em(5) em(10);
+  border-radius: $base-border-radius;
+  -webkit-transition: 0.3s ease-in-out;
+  -moz-transition: 0.3s ease-in-out;
+  -o-transition: 0.3s ease-in-out;
+  transition: 0.3s ease-in-out;
+
+  & a {
+    color: $white;
+  }
+
+  &:hover {
+    background-color: rgba($white, 0.2);
+    cursor: pointer;
+  }
+
+  &-icon {
+    background: image-url("twitter-icon.png");
+    background-repeat: no-repeat;
+    background-size: em(19);
+    display: inline-block;
+    height: em(15);
+    margin-left: em(10);
+    width: em(20);
+  }
+}
+
+.compliment {
+  color: $white;
+  margin-top: 15%;
+  text-align: center;
+
+  &-refresh {
+    text-align: center;
+
+    a {
+      border-bottom: 1px solid $white;
+      color: $white;
+      opacity: 0.8;
```

Comment:
Properties should be ordered border-bottom, color, display, line-height, margin, opacity, text-shadow